### PR TITLE
Android SDK installation instructions point to 3.+

### PR DESCRIPTION
**Pull request by giorgiomartini0**
Instead of specific version. All changes on `3.x.x` will be non-breaking.

### CODE REVIEW

File: docs/sdks/client-sdks/android.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Eppo's open source Android SDK can be used for both feature flagging and experim
 You can install the SDK using Gradle by adding to your `build.gradle` file:
 
 ```groovy
-implementation 'cloud.eppo:android-sdk:3.2.0'
+implementation 'cloud.eppo:android-sdk:3.+'
```
Comment (felipecsl) [inline docs/sdks/client-sdks/android.md:16]: FYI it's not recommended to use `+` syntax on Gradle as it leads to non-deterministic behavior with unexpected changing library versions. I know this is a different convention than what folks usually do for NPM/JS ecossystem, for example, but on JVM world people are a lot more conscious of this stuff. Just wanted to call that out

Comment (felipecsl) [reply docs/sdks/client-sdks/android.md:16]: Just a data point:
image
https://developer.vonage.com/en/blog/the-right-way-of-using-gradle-dynamic-dependencies

Comment (giorgiomartini0) [reply docs/sdks/client-sdks/android.md:16]: Good callout - I didn't know. Reverting to point explicitly to the latest version.

Comment (felipecsl) [reply docs/sdks/client-sdks/android.md:16]: thank you 🙏🏽

**Pull request merged**